.github/workflows/enforce-develop-to-main.yml

```
@@ -0,0 +1,19 @@
+on:
+  pull_request_target:
+    branches: [main]
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  enforce:
+    name: require develop as source
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fail if head branch is not develop
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)